python-main.js

```diff
@@ -1792,7 +1805,7 @@ function web_editor(config) {
             var WebUSBUnavailable = function() {
                 var links = {};
                 links[config['translate']['webusb']['err']['find-more']] = 'help.html#WebUSB';
-                modalMsg('WebUSB', config['translate']['webusb']['err']['unavailable'], links);
+                modalMsg('WebUSB', config['translate']['webusb']['err']['unavailable'], links)
```

Comment:
The semicolon could be reverted back to reduce the diff.

Reply:
fixed in https://github.com/bbcmicrobit/PythonEditor/pull/310/commits/7ebd2794a1f8ef0cd823a3ad67df4b139a80bdce